BitFaster.Caching.UnitTests/Lru/DiscretePolicyTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using BitFaster.Caching.Lru;
+using FluentAssertions;
+using Xunit;
+
+namespace BitFaster.Caching.UnitTests.Lru
+{
+    public class DiscretePolicyTests
+    {
+        private readonly TestExpiryCalculator<int, int> expiryCalculator;
+        private readonly DiscretePolicy<int, int> policy;
+
+        private static readonly ulong stopwatchDelta = (ulong)StopwatchTickConverter.ToTicks(TimeSpan.FromMilliseconds(20));
+        private static readonly ulong tickCountDelta = (ulong)TimeSpan.FromMilliseconds(20).ToEnvTick64();
+
+        private static readonly TimeSpan defaultTimeToExpire = TimeSpan.FromSeconds(10);
+
+        public DiscretePolicyTests() 
+        {
+            expiryCalculator = new TestExpiryCalculator<int, int>();
+            expiryCalculator.ExpireAfterCreate = (_, _) => defaultTimeToExpire;
```

Comment:
This should be the default for the test calculator, so its not repeated. Default time to expire can be there as well.